mixins/_icons.scss

```diff
@@ -95,18 +102,21 @@
     @if $symbol == 'info' {
         content: "\49";
     }
-    @if $symbol == 'left' {
-        content: "\3c";
-    }
     @if $symbol == 'left-big' {
         content: "\2039";
     }
     @if $symbol == 'left-sign' {
         content: "\25c1";
     }
+    @if $symbol == 'left' {
+        content: "\3c";
+    }
     @if $symbol == 'less-sign' {
         content: "\229d";
     }
+    @if $symbol == 'lock' {
+        content: "\6c";
+    }
```

Comment:
Didn't have a mixin for this before, even though it was in the font files. I've added it just in case.